src/current/v23.1/backup.md

```diff
@@ -143,19 +143,28 @@ The `system.users` table stores your users and their passwords. To restore your
+
+Unlike a normal [read transaction](architecture/reads-and-writes-overview.html#read-scenario) that will block until any uncommitted writes it encounters are resolved, a backup job will be allotted a fixed amount of CPU time to read the required keys and values. Once the backup's read request has exhausted this time, the backup will resume once it has been allocated more CPU time. This process allows for other requests, such as foreground SQL traffic to continue, almost unaffected, because there is a cap on how much CPU a backup job will take.
+
+You can monitor your cluster's [admission control system](admission-control.html) on the [Overload dashboard](ui-overload-dashboard.html). To monitor your backup jobs, refer to the [Backup and Restore Monitoring](backup-and-restore-monitoring.html) page.
+
+For a more technical explanation of elastic CPU, refer to the [Rubber control theory on the Go scheduler](https://www.cockroachlabs.com/blog/rubbing-control-theory/) blog post.
 
 We recommend always starting backups with a specific [timestamp](timestamp.html) at least 10 seconds in the past. For example:
 
 ~~~ sql
-> BACKUP...AS OF SYSTEM TIME '-10s';
+BACKUP...AS OF SYSTEM TIME '-10s';
 ~~~
 
-This improves performance by decreasing the likelihood that the `BACKUP` will be [retried because it contends with other statements/transactions](transactions.html#transaction-retries). However, because `AS OF SYSTEM TIME` returns historical data, your reads might be stale. Taking backups with `AS OF SYSTEM TIME '-10s'` is a good best practice to reduce the number of still-running transactions you may encounter, since the backup will take priority and will force still-running transactions to restart after the backup is finished.
+This improves performance by decreasing the likelihood that the `BACKUP` will be [retried because it contends with other statements/transactions](transactions.html#transaction-retries). However, because [`AS OF SYSTEM TIME`](as-of-system-time.html) returns historical data, your reads might be stale. Taking backups with `AS OF SYSTEM TIME '-10s'` is a best practice to reduce the number of still-running transactions you may encounter, since the backup will take priority and will force still-running transactions to restart after the backup is finished.
 
-`BACKUP` will initially ask individual ranges to backup but to skip if they encounter an intent. Any range that is skipped is placed at the end of the queue. When `BACKUP` has completed its initial pass and is revisiting ranges, it will ask any range that did not resolve within the given time limit (default 1 minute) to attempt to resolve any intents that it encounters and to **not** skip. Additionally, the backup's transaction priority is then set to `high`, which causes other transactions to abort until the intents are resolved and the backup is finished.
```

Comment:
I assume removing this end sentence is correct with the integration of elastic CPU, but perhaps not?

Reply:
The elastic CPU limiter is independent of the behaviour described in this sentence. I would not remove the sentence but instead rewrite it to:

Additionally, the backup's read transaction priority is eventually set to `high`. This will result in contending transactions being pushed and retries at a higher timestamp.